api/document/js/parse-doc.ts

```diff
@@ -24,8 +25,16 @@ export function convertContent(content, marks: Mark[]): Node[] {
   return flatten(nestedChildNodes);
 }
 
-const NODE_TYPE_CONVERTERS = {
+
+type NodeConverterMap = {
+  [nodeName: string]: (node: ApiNode) => Node,
+};
```

Comment:
In the future, I'd like to improve upon this to actually require that `nodeName` be a valid API node type.